core/src/main/java/org/jenkins/ui/icon/IconSet.java

```diff
@@ -517,24 +525,33 @@ private static void initializeSVGs() {
         images.add("warning");
         images.add("document-properties");
 
-        Map<String, String> materialIcons = new HashMap<>();
-        materialIcons.put("help", "svg-sprite-action-symbol.svg#ic_help_24px");
-
         for (Map.Entry<String, String> size : sizes.entrySet()) {
             for (String image : images) {
                 icons.addIcon(new Icon("icon-" + image + " " + size.getKey(),
                         "svgs/" + image + ".svg", size.getValue()));
             }
+        }
+    }
 
-            for (Map.Entry<String, String> imageEntry : materialIcons.entrySet()) {
-                icons.addIcon(new Icon(
-                        "icon-" + imageEntry.getKey() + " " + size.getKey(),
-                                "material-icons/" + imageEntry.getValue(),
-                                size.getValue(),
-                                IconFormat.EXTERNAL_SVG_SPRITE
-                        )
-                );
-            }
+    /**
+     * This is a temporary function to replace Tango icons across Jenkins and plugins with
+     * appropriate Jenkins Symbols
+     *
+     * @param tangoIcon A tango icon in the format 'icon-* size-*', e.g. 'icon-gear icon-lg'
+     * @return a Jenkins Symbol (if one exists) otherwise null
+     */
+    public static String tryTranslateTangoIconToSymbol(String tangoIcon) {
```

Comment:
no need to be part of the API I think? ) internal use only
```suggestion
    @Restricted(NoExternalUse.class)
    public static String tryTranslateTangoIconToSymbol(String tangoIcon) {
```